### PR TITLE
chore: switch to prettier

### PR DESCRIPTION
**Pull request by SuperchupuDev**
## Summary
while biome is good, some users may think prettier is better, which is why this pr switches to prettier

TODO:
- update ci to build prettier from source

### CODE REVIEW

**Comment by SuperchupuDev** [comment]
april fools

**Comment by MoritzHeld** [comment]
LGTM💪

**Comment by Quentindeve** [comment]
lgtm

**Comment by dBianchii** [comment]
Yeah, look at that massive -14 LOC win.
Let’s goo

**Comment by SuperchupuDev** [comment]
it turns out unzipping prettier is pretty slow. should i install xz and
decompress it with that?

**Comment by Netail** [comment]
The audacity

**Comment by jacobdalamb** [comment]
LGTM

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (DaniGuardiola) [inline .prettierrc:1]: needs more options

**Review by DaniGuardiola — CHANGES_REQUESTED** [review]
important nit

File: .prettierrc (hunk 1 above)
Comment (SuperchupuDev) [reply .prettierrc:1]: thank you. added now

**Pull request closed without merging**